src/Neo/SmartContract/Native/OracleContract.cs

```diff
@@ -202,7 +204,7 @@ private async ContractTask Request(ApplicationEngine engine, string url, string
             if (urlSize > MaxUrlLength)
                 throw new ArgumentException($"The url bytes size({urlSize}) cannot be greater than {MaxUrlLength}.");
 
-            var filterSize = filter.GetStrictUtf8ByteCount();
+            var filterSize = filter == null ? 0 : filter.GetStrictUtf8ByteCount();
```

Comment:
That's the bug, if null should continue, this condition was removed, and it's allowed a null filter

Reply:
Removed here:
https://github.com/neo-project/neo/pull/3862/files#diff-f3b73519c694bc49d45cd00df28c9559a02fc27c569bb53e9edd234b782ebf1d